hc/accounts/views.py

```diff
@@ -156,9 +162,11 @@ def profile(request):
         elif "update_reports_allowed" in request.POST:
             form = ReportSettingsForm(request.POST)
             if form.is_valid():
-                profile.reports_allowed = form.cleaned_data["reports_allowed"]
+                profile.reports_frequency = form.data["reports_allowed"]
```

Comment:
Why aren't you cleaning the data?

Reply:
It was giving me a problem but I should find a way around that

Reply:
oooh I stopped using the function because "cleaned_data" function is customised to make the input boolean.

Reply:
You're doing something wrong in this case. `cleaned_data` should return data that's safe to store in the db, in order to avoid stuff like XSS

Reply:
@jimmykamau I have defined another form which allows the input am expecting from my forms. So I have been able to use cleaned data now